tests/common/mod.rs

```diff
@@ -5,6 +5,110 @@ use web_sys::{
     wasm_bindgen::JsValue,
 };
 
+async fn time_it<R>(cb: impl std::future::Future<Output = R>) -> (f64, R) {
+    let now = {
+        use web_sys::{
+            js_sys::{global, Reflect},
+            wasm_bindgen::JsCast,
+            Performance,
+        };
+
+        let performance = Reflect::get(&global(), &"performance".into())
+            .unwrap()
+            .unchecked_into::<Performance>();
```

Comment:
Can this make use of the same logic as [`Factory::get()`](https://github.com/ekleog/indexed-db/blob/fe645fae2a91a48a0af6126611c2031d59db0ad1/src/factory.rs#L27-L33)?
```rust
        let indexed_db = if let Some(window) = web_sys::window() {
            window.indexed_db()
        } else if let Ok(worker_scope) = js_sys::global().dyn_into::<WorkerGlobalScope>() {
            worker_scope.indexed_db()
        } else {
            return Err(crate::Error::NotInBrowser);
        };
```

OTOH it's probably not a big deal because this is only tests, but it'd feel better to me to have proper type safety guaranteed by `web-sys`, rather than unchecked casts that we do ourselves, even if they're certainly correct :smile:

Reply:
That's cleaner !